.github/workflows/trivy.yml

```diff
@@ -0,0 +1,10 @@
+name: Call Update Trivy Cache
```

Comment:
Sorry a little nitpicking. Correct me if I'm wrong but this workflow doesn't trigger trivy scan, it just cached the DB, right? 
So maybe let's rename it to trivy-db-cache of something like this?

Reply:
changed it